api/main/models/observation.py

```diff
@@ -90,8 +90,10 @@ def __init__(self, init_dict):
                 Note: not all are needed, many can be initialized initially
                       as null.
         """
-
-        self.set_attrs(init_dict)
+        if type(init_dict) is dict:
+            self.set_attrs(init_dict)
+        else:
+            raise ValueError( 'init_dict must be iterable like a dictionary' )
```

Comment:
I *think* that this can be replaced by typing the in parameter as a dict: `init_dict` could/should become something like `init_cols: dict`. This is perfect for now